### ISSUE TITLE
Additional Shortcuts for Tag Manager

### ISSUE THREAD

**Issue by TheElo** [hydrusnetwork/hydrus#657]
There some shortcuts missing to make the tag manager more keyboard user friendly and allow them to fully manipulate the tag manager without mouse interaction. In order of subjective importance from top/best to buttom/just here for completeness

Suggested Tags:
"thorough" and "medium" have no shortcuts - after adding tags from recent/suggested/typed/shortcuts it helps to update the suggested tag list. This is currently not possible without a mouse action or closing and reopening the tag manager or moving a file forward then back.

Current Tags:
"focus on current tag list" is missing, there is no way for the user to remove tags manually from the list to correct false tags or mistakes.
"copy tags"
"paste tags" 
("remove all/selected tag")

File Lookup:
"fetch tags" a shortcut for the actual action of looking up files is missing.
"add all" a shortcut add all tags is missing
"focus on lookup tag list" there is no shortcut to focus on lookup list

Recent Tags:
"clear"

a lot of the rest can be tabbed to afaik
